usaspending_api/download/tests/integration/test_download_awards.py

```diff
@@ -122,9 +122,23 @@ def test_download_awards_without_columns(client, download_test_data):
     assert ".zip" in resp.json()["url"]
 
 
+@pytest.mark.django_db
+def test_tsv_download_awards_without_columns(client, download_test_data):
+    download_generation.retrieve_db_string = Mock(return_value=generate_test_db_connection_string())
+    resp = client.post(
+        "/api/v2/download/awards/",
+        content_type="application/json",
+        data=json.dumps({"filters": {"award_type_codes": []}, "columns": []}),
```

Comment:
Going off the contract, shouldn't the tests have `file_format` in the data that is passed through the post?

```suggestion
        data=json.dumps({"filters": {"award_type_codes": []}, "columns": [], "file_format": "tsv"}),
```

Reply:
Isn't the field marked as optional?

Reply:
Yes, but since this test is testing that it works wouldn't you want to actually pass it along? All the other download tests are already testing that it works fine without it.

Reply:
AH, I understand